tests/conftest.py

```diff
@@ -0,0 +1,40 @@
+import datetime
+
+import pytest
+from app.database.database import Base, SessionLocal, engine
+from app.database.models import Event, User
+from app.main import app
+from faker import Faker
+from fastapi.testclient import TestClient
+from hypothesis import given
+from hypothesis.strategies import emails
+
+
+@pytest.fixture
+def client():
+    return TestClient(app)
+
+
+@pytest.fixture
+def sessions():
+    Base.metadata.create_all(bind=engine)
+    sessions = SessionLocal()
+    yield sessions
+    Base.metadata.drop_all(bind=engine)
```

Comment:
This is seems dangerous to me - it look likes it can delete the production database if run on prod.